v1.3

bellyillish opened the pull request:
WIP

bellyillish committed:
optimize combat hit and hear callbacks

bellyillish committed:
patches for surge_rush and itms_manager

bellyillish committed:
add callbacks for opening and closing UI

bellyillish committed:
remove nil from ends of dltx

bellyillish committed:
fix error with grok_companions_no_ammo

bellyillish committed:
fix followers looking away when approaching

bellyillish committed:
fix pointsAlongAxis() doubling point at 0/360

bellyillish committed:
universal stuck animation fixes for state_mgr.set_state()

bellyillish committed:
improved state/behavior persistence

bellyillish committed:
surge behavior

bellyillish committed:
fix bug with companion spacing

bellyillish committed:
improve companion checks in axr_beh

bellyillish committed:
reload fix when crouching

bellyillish commented:
@akzar-dev obviously no pressure but if you _are_ still interested in helping with translation, I'm getting ready to push this as v1.3. There is one new section in MCM for surge options. That should be only change(s).

bellyillish committed:
fix tabs

bellyillish committed:
add missing comments

akzar-dev commented:
Hey @bellyillish , if it can wait till tomorrow - would be happy to help out!

bellyillish commented:
@akzar-dev of course, no rush at all. I don't plan to release this for at least a couple more days I have more testing I want to do

akzar-dev commented:
@bellyillish https://github.com/bellyillish/useful-idiots/pull/15 here's the translation for 1.3 surge features (plus small fix for forgotten "Loners" translation in one of the menus)

akzar-dev committed:
chore: russian translation fixes for 1.3 (#15)

bellyillish committed:
final cleanup

bellyillish committed:
translations for new mcm option

bellyillish commented:
@akzar-dev I'm sorry, I added one more thing -- an MCM checkbox. I just pushed a commit with the new labels, but this is the snippet I added:
```
  <string id="ui_mcm_idiots_options_splitSquads">
    <text>Split Each Companion Into Own Squad</text>
  </string>
  <string id="ui_mcm_idiots_options_splitSquads_desc">
    <text>Lets companions move individually between maps. Otherwise they may not stay/follow as expected or may teleport to far-away positions when transitioning between maps. Squads cannot be restored once split.</text>
  </string>
```

bellyillish committed:
add mcm option to disable splitting squads

akzar-dev commented:
@bellyillish no prob, I’ll translate it tonight or tomorrow morning

bellyillish committed:
removed unneeded combat patch

bellyillish committed:
updated readme

akzar-dev commented:
@bellyillish here it is https://github.com/bellyillish/useful-idiots/pull/16

akzar-dev committed:
chore: translate split squad mcm stuff (#16)
